### PR TITLE
Fixed two bugs

### PR DESCRIPTION
**Pull request by YowFung**
Fixed following bugs (#8 ):
- Unhandled Exception: A ValueNotifier<dynamic> was used after being disposed.
- Unhandled Exception: Looking up a deactivated widget's ancestor is unsafe.

### CODE REVIEW

**Comment by jaumard** [comment]
Hey :) not sure try/catch is a right fix here ^^ I'll try to dig more into this this week. 
I'm thinking on a cleaner solution I'll try to implement ! But anyway it's helpful as now I know where is the crash happening so thanks ! Keeping this open because if my solution doesn't work I'll merge this one :)

File: example/ios/Flutter/flutter_export_environment.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```
Comment (jaumard) [inline example/ios/Flutter/flutter_export_environment.sh:2]: should not be committed, please remove

**Comment by YowFung** [comment]
Thanks, please.

**Comment by jaumard** [comment]
Done on master, will publish soon, thanks

**Pull request closed without merging**